build.xml

```diff
@@ -2034,27 +2061,36 @@
                        <br><a href="http://jmri.org/Copyright.html">Additional information on copyright, trademarks and licenses is linked here.</a>
                 ]]>
             </bottom>
+
+            <!-- <link href="http://java.sun.com/products/javacomm/reference/api/"/> redirected to next line -->
+            <link href="https://docs.oracle.com/cd/E17802_01/products/products/javacomm/reference/api/" />
+                        
+            <!-- <link href="https://static.javadoc.io/com.fasterxml.jackson.core/jackson-databind/2.9.8/"/>  redirected to next line -->
+            <link href="https://www.javadoc.io/doc/com.fasterxml.jackson.core/jackson-databind/2.9.8/" />
```

Comment:
This needs to remain as an offline reference as the processing of the `javadoc-uml` target doesn't cope with the way Google Tag Manager code is embedded in items hosted at javadoc.io. See https://github.com/JMRI/JMRI/pull/8068/files#diff-2cccd7bf48b7a9cc113ff564acd802a8L2041 for (deleted) reasoning.

Reply:
With this as-is, the [Generate Website](https://builds.jmri.org/jenkins/job/website/job/generate-website/) job on Jenkins fails.

Reply:
Not sure what's meant by "offline reference" here.  The only change was to the URL, not the structure of the `link` element.  The new URL was taken from the message when running Javadoc under Java 11, which says that it was a redirect.

Reply:
The issue is that for `javadoc-uml` to process, we need to set any javadoc.io hosted documents as though they were offline. This is due to them injecting some Google Tag Manager code in their published JavaDoc pages which the `javadoc-uml` target attempts to parse (when not marked as being offline) thereby dumping. Bit of a dirty hack, but it gets that target to build. Other javadoc targets are not affected.

Reply:
I'll work up a PR later to illustrate what I mean.

Reply:
See #8078 for possible solution